tools/ci/checks/run_pylint_checks.sh

```diff
@@ -4,6 +4,7 @@ DOCKER_TAG=${DOCKER_TAG:-latest}
 DOCKER_IMAGE=${DOCKER_IMAGE:-mesosphere/dcos-commons:${DOCKER_TAG}}
 
 docker run --rm -t \
+    -e PYTHONPATH="$PYTHONPATH" \
```

Comment:
Nice. I was going to report pylint erred out on `master` with 
```
[18:49:29]	[Step 5/6] Running pylint on 2 files:
[18:49:29]	[Step 5/6] No config file found, using default configuration
[18:49:30]	[Step 5/6] ************* Module tests.client
[18:49:30]	[Step 5/6] E:  7, 0: Unable to import 'sdk_auth' (import-error)
[18:49:30]	[Step 5/6] E:  8, 0: Unable to import 'sdk_cmd' (import-error)
[18:49:30]	[Step 5/6] E:  9, 0: Unable to import 'sdk_install' (import-error)
[18:49:30]	[Step 5/6] E: 10, 0: Unable to import 'sdk_marathon' (import-error)
```

Reply:
Thanks for keeping an eye on this!